datamade/just-spaces
# Display ACS data for selected questions in DAD

## Overview

Closes #100.

Update the Data Analysis Designer to allow the user to add ACS data to charts for specific survey questions.

## Testing Instructions

* Create a survey that has all preset Intercept questions
    * Add a few custom Intercept questions so that you can make sure that the DAD doesn't allow you to compare them to ACS data
* Create a survey that has the following preset Observational questions:
    *  Age, all detail levels
    * Gender
    * Add a few custom Observational questions as well
* Publish your surveys and run them a few times with dummy data
* Navigate to the submitted survey view for each survey and confirm:
    * You are shown ACS geography options for qualifying questions
    * You can select different combinations of ACS geographies and see the data displayed on the chart
    * You are not shown ACS geography options for the custom questions you created
* Save some charts with ACS data, and confirm that when you reload the page they load with the data intact
